tck/features/clauses/create/Create2.feature

```diff
@@ -0,0 +1,296 @@
+      | +nodes         | 1 |
+      | +relationships | 1 |
+
+  Scenario: [10] Creating a single self loop on a existing node
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:Root)
+      """
+    When executing query:
+      """
+      MATCH (root:Root)
+      CREATE (root)-[:LINK]->(root)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +relationships | 1 |
+
+  Scenario: [11] Creating a single relationship and an end node on a existing starting node
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:Begin)
+      """
+    When executing query:
+      """
+      MATCH (x:Begin)
+      CREATE (x)-[:TYPE]->(:End)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 1 |
+      | +relationships | 1 |
+      | +labels        | 1 |
+    When executing control query:
+      """
+      MATCH (x:Begin)-[:TYPE]->(y:End)
+      RETURN x, y
+      """
+    Then the result should be, in any order:
+      | x        | y      |
+      | (:Begin) | (:End) |
+
+  Scenario: [12] Creating a single relationship and an starting node on a existing end node
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:End)
+      """
+    When executing query:
+      """
+      MATCH (x:End)
+      CREATE (:Begin)-[:TYPE]->(x)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 1 |
+      | +relationships | 1 |
+      | +labels        | 1 |
+    When executing control query:
+      """
+      MATCH (x:Begin)-[:TYPE]->(y:End)
+      RETURN x, y
+      """
+    Then the result should be, in any order:
+      | x        | y      |
+      | (:Begin) | (:End) |
+
+  Scenario: [13] Creating a single relationship with a property
+    Given any graph
+    When executing query:
+      """
+      CREATE ()-[:R {num: 42}]->()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+      | +properties    | 1 |
+
+  Scenario: [14] Creating a single relationship with a property and returning it
+    Given any graph
+    When executing query:
+      """
+      CREATE ()-[r:R {num: 42}]->()
+      RETURN r.num AS num
+      """
+    Then the result should be, in any order:
+      | num |
+      | 42  |
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+      | +properties    | 1 |
+
+  Scenario: [15] Creating a single relationship with two properties
+    Given any graph
+    When executing query:
+      """
+      CREATE ()-[:R {id: 12, name: 'foo'}]->()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+      | +properties    | 2 |
+
+  Scenario: [16] Creating a single relationship with two properties and returning them
```

Comment:
Same as for comment in Create 1, line 86

Reply:
see reply to comment for line 86